derive/src/to_delegate.rs

```diff
@@ -169,9 +171,7 @@ impl RpcMethod {
 			special_args.iter().find(|(_,sty)| sty == ty).is_none());
 
 		let tuple_fields : &Vec<_> =
-			&(0..param_types.len() as u8)
-				.map(|x| ident(&((x + 'a' as u8) as char).to_string()))
-				.collect();
+			&(TUPLE_FIELD_NAMES[0..param_types.len()].iter().map(|name| ident(name)).collect());
```

Comment:
I'm actually not sure about this, since it doesn't support more than 26 items — but I guess the results of having more in the previous edition might've also be unexpected?
@ascjones what do you think?